cheroot/errors.py

```diff
@@ -36,19 +35,29 @@ def plat_specific_errors(*errnames):
 
 socket_errors_to_ignore = plat_specific_errors(
     'EPIPE',
-    'EBADF', 'WSAEBADF',
```

Comment:
Things like this are paired because of their related semantic meaning. These lines mustn't be messed up into something that doesn't relay this connection. Alternatively, they'd have to be moved to separate variables with names and unpacked here.